src/Gateway/GatewayModule.php

```diff
@@ -833,10 +834,9 @@ public function addBirthdateWhenRest($arrayContext)
                 $context->order->update_meta_data('billing_birthdate', $billingBirthdate);
                 $context->order->save();
             } else {
-                $message = __('Please introduce a valid birthdate number.', 'mollie-payments-for-woocommerce');
                 throw new RouteException(
                     'woocommerce_rest_checkout_process_payment_error',
-                    $message,
+                    esc_html__('Please introduce a valid birthdate number.', 'mollie-payments-for-woocommerce'),
```

Comment:
Just as some context: This particular sniff is a bit controversial: https://github.com/WordPress/WordPress-Coding-Standards/issues/2374

But we'll keep it since we probably do not control where/how the exception is handled/rendered - and escaping is not a huge deal here.